src/consensus/params.h

```diff
@@ -11,6 +11,15 @@
 
 namespace Consensus {
 
+enum BuriedDeployment
```

Comment:
prefer enum class to prevent leaking names.

Reply:
Also helps with avoiding overlap with SignalledDeployment.

Reply:
Leaking the names is the point -- otherwise you'd need to change every use from `Consensus::DeploymentPos::SEGWIT` to `Consensus::BuriedDeployment::SEGWIT`. With C++20, you will be able to specify `enum class DeploymentPos { .. }; using enum DeploymentPos;` but that's a while away.